src/views/v2/components/events-bar.php

```diff
@@ -41,15 +41,11 @@
 		<?php $this->template( 'components/events-bar/search-button' ); ?>
 
 		<div
-			class="tribe-events-c-events-bar__search-filters-container"
-			id="tribe-events-search-filters-container"
-			data-js="tribe-events-search-filters-container"
+			class="tribe-events-c-events-bar__search-container"
```

Comment:
This change will break back-compat with existing CSS modifications, was this intentional?

Reply:
To answer your question I think B20.09 will be a feature release for both TEC and Filterbar